src/frequenz/sdk/actor/power_distributing/_battery_pool_status.py

```diff
@@ -123,6 +123,17 @@ def __init__(
 
         self._task = asyncio.create_task(self._run())
 
+    async def join(self) -> None:
+        """Await for the battery pool, and return when the task completes.
+
+        It will not close the program until BatteryPool is working.
```

Comment:
I guess this means: `It will not terminate the program while BatteryPool is working.` ?

Reply:
Done